specifications/move_adapter/README.md

```diff
@@ -313,6 +317,17 @@ transaction sender's account is frozen. If so, the status code is set to
 in the `SignedTransaction`) matches the authentication key in the sender's
 account. If not, validation fails with an `INVALID_AUTH_KEY` status code.
 
+* If secondary signers exist for the transaction, perform the following checks:
```

Comment:
This text looks OK, but earlier in this section, there needs to be some explanation of the different prologue function name that is used for multi-agent txns.